### ISSUE TITLE
Goal Progress Issue

### ISSUE THREAD

**Issue by Rhettica** [lynchjames/obsidian-writing-goals#35]
Hi,

I recently turned on the 'Display Goal Stats' option and nothing appears on the sidebar.
When I hover over a certain part of the screen, the text appears, but the bars and background don't.
I changed the theme and the background box appeared so perhaps this issue could be fixed with a background colour option but it doesn't appear to be counting past progress. 

Could you please assist with this?

<img width="326" alt="Screenshot 2023-09-16 at 12 00 09 pm" src="https://github.com/lynchjames/obsidian-writing-goals/assets/67592285/115e6bda-0423-409e-bda3-6608cfe56d65">
<img width="326" alt="Screenshot 2023-09-16 at 12 00 30 pm" src="https://github.com/lynchjames/obsidian-writing-goals/assets/67592285/12d981fc-35d5-49f9-b293-c97be0cbd506">

**Comment by lynchjames** [comment]
Are there any entries for the file/folder path in `goal-history.json` in the WG plugin folder?

**Comment by danieltomasz** [comment]
@lynchjames  it seems to be a problem with a theme, @Rhettica   which theme are you using when you experience the problems?

**Comment by lynchjames** [comment]
@danieltomasz I didn't think it was theme-related because in the screenshots you can see the outer ring and "word goal" text against the background which would be the same color as the stats bar chart.

**Comment by Rhettica** [comment]
> Are there any entries for the file/folder path in `goal-history.json` in the WG plugin folder? Also, do you see any past progress recorded in the Stats view?

Yes, there are several entries in that file but i will test it again tonight by adding more words and seeing if it works tomorrow, just in case.

**Comment by Rhettica** [comment]
> @lynchjames it seems to be a problem with a theme, @Rhettica which theme are you using when you experience the problems?

I'm using the AnuPpuccin theme. I also tested with the Dracula and default themes.

**Comment by Rhettica** [comment]
Ok, i think i fixed it and i think it was my fault. I started playing with the goal-history.json file and realised that it didn't adjust after I deleted my words. 

For example, If I wrote 500 words yesterday and deleted them because I didn't like them. It still showed the date when I hovered over where the bars should be so I thought it wasn't working and that something should be there because I wasn't keeping track of the words myself. After manually changing the words in the file, it looks like this screenshot. I think it's working now. Really sorry about that. 

<img width="321" alt="Screenshot 2023-09-18 at 8 17 15 pm" src="https://github.com/lynchjames/obsidian-writing-goals/assets/67592285/16875d44-a99b-4c17-9962-ba629cc3e1ae">
